python_envs/environment.yaml

```
@@ -0,0 +1,51 @@
+channels:
+- conda-forge
+dependencies:
+- python>=3.12
```

I think it is more robust to clearly define the python version. Otherwise conda/mamba may or may not use 3.12 depending on the python version in the base environment (at least that's my current guess...)
```suggestion
- python==3.12
```